### PR TITLE
fix transfer error when inspector and TLS is on, but the conn is Plaintext

### PR DESCRIPTION
**Pull request by taoyuanyuan**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Sign the CLA
Make sure you have signed the [CLA](https://www.clahub.com/agreements/alipay/sofa-mosn)

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, 
details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases.
And you need to show the ut result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/server/handler.go [hunk 1]
```diff
@@ -402,7 +402,8 @@ func (al *activeListener) OnAccept(rawc net.Conn, useOriginalDst bool, oriRemote
 				rawf, _ = tc.File()
 			}
 		}
-		if al.tlsMng != nil {
+		// if ch is not nil, the conn has been initialized in func transferNewConn
+		if al.tlsMng != nil && ch == nil {
```
Comment (nejisama) [inline pkg/server/handler.go]: 这个判断是否可靠？ 我看UseOriginalDst这个函数也会传递ch

Comment (taoyuanyuan) [reply pkg/server/handler.go]: 这个chan在连接迁移的时候才会有，连接迁移的场景conn都是设置好了的了，不用再去tlsMgr初始化
